compiler/ballerina-parser/src/test/resources/expressions/xml-template/xml_template_source_32.bal

```diff
@@ -0,0 +1,11 @@
+function foo() {
+    xml a = xml `<![CDATA[first text ]]> second text ]]>`;
```

Comment:
this is not a negative syntax test case right. Can move to the source_31.bal?

Reply:
The idea here is to test that CDATA Sections cannot contain `]]>`. Since `second text ]]>` is again a valid XML text the test will not contain any diagnostic.